packages/flame/lib/src/game/game.dart

```diff
@@ -216,6 +216,16 @@ abstract class Game {
   /// - GameWidget
   /// - [Game.overlays]
   final overlays = ActiveOverlaysNotifier();
+
+  /// Use this method in case you need to project a coordinate received by a
+  /// gesture/pointer event to your game coordinate system.
+  ///
+  /// By default this just returns the same received coordinate transformed to
+  /// a Vector2, override this to add addtional logic to that projection
+  Vector2 projectCoordinates(Offset value) => value.toVector2();
+
+  // TODO(erickzanardo) Luan, help me here on BaseGame logic
+  Vector2 scaleCoordinate(Offset value) => value.toVector2();
```

Comment:
@luanpotter gonna need your help on how to correctly implement this on `BaseGame`